### PR TITLE
add Kind to local-machine solutions

### LINKED ISSUE #13176: add Kind to local-machine solutions
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

**Problem:**

[kind](https://github.com/kubernetes-sigs/kind) is a tool for running local Kubernetes clusters using Docker container "nodes".
We can use it to create multi-control-plane or multi-node Kubernetes cluster.

**Page to Update:**
https://kubernetes.io/docs/setup/pick-right-solution/

### PR DESCRIPTION
**Pull request by tao12345666333**
Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>

fix #13176 

/cc @neolit123

### CODE REVIEW

**Comment by tao12345666333** [comment]
https://deploy-preview-13177--kubernetes-io-master-staging.netlify.com/docs/setup/pick-right-solution/#community-supported-tools

File: content/en/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ a Kubernetes cluster from scratch.
 
 * [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) is a multi-node (while minikube is single-node) Kubernetes cluster which only requires a docker daemon. It uses docker-in-docker technique to spawn the Kubernetes cluster.
 
+* [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind) is a tool for running local Kubernetes clusters using Docker container "nodes". It is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md). You can use it to create multi-control-plane(while kubeadm-dind is single-control-plane) or multi-node Kubernetes cluster.
```
Comment (neolit123) [inline content/en/docs/setup/pick-right-solution.md]: best to change the last sentence to:
> You can use it to create multi-node or multi-control-plane Kubernetes clusters.

Comment (BenTheElder) [reply content/en/docs/setup/pick-right-solution.md]: +1, also we can drop the conformance tests note, I've meant to do that upstream. We test other things with kind. `It is primarily designed for testing Kubernetes 1.11+.` is sufficient on it's own, the conformance bit is just noise now.

Comment (tao12345666333) [reply content/en/docs/setup/pick-right-solution.md]: @neolit123 @BenTheElder Thanks. :+1:  I have updated.

**Comment by karthequian** [comment]
Kind already exists in the `Ecosystem Tools` section. If it's being moved to the community supported tools sections, we should perhaps remove from the `ecosystem tools`. If it's in both places, at least let's make the text the same in both sections.

**Comment by tao12345666333** [comment]
@karthequian  Thanks. I have removed the second section.

**Pull request merged**